### PR TITLE
Add a section about local testing

### PR DESCRIPTION
**Pull request by michadu**
A short section about local testing with the bamboo docker containers.

### CODE REVIEW

File: Documentation/BugfixingAZ/Index.rst [hunk 1]
```diff
@@ -79,6 +79,56 @@ If you should encounter any problems or have questions, talk to us on Slack_ in
    work against this standard, install PHP Codesniffer, ask us if you need any assistance.
 
 
+Test the code
+=============
+
+.. highlight:: shell
+
+It is a good idea to test the TYPO3 core with your fix to be shure that the automatic tests, that are
```
Comment (liayn) [inline Documentation/BugfixingAZ/Index.rst]: shure -> sure

File: Documentation/BugfixingAZ/Index.rst [hunk 2]
```diff
@@ -79,6 +79,56 @@ If you should encounter any problems or have questions, talk to us on Slack_ in
...
+All you need locally for this is a running docker setup.
+
+You have to load the php container you need from https://bitbucket.typo3.com/projects/T3COM/repos/bamboo-remote-agent/browse
+There are a lot of containers for testing with different PHP versions and different databases
+(for the functional tests).
+As an example, if you want to run the unit tests for current master you have to load the php72 container::
+
+   docker pull typo3gmbh/php72:latest
+
+Than you have to start the container. It will end up with a bash open::
```
Comment (liayn) [inline Documentation/BugfixingAZ/Index.rst]: Than -> Then

File: Documentation/BugfixingAZ/Index.rst [hunk 3]
```diff
@@ -79,6 +79,56 @@ If you should encounter any problems or have questions, talk to us on Slack_ in
...
+database credentials inside of the ``Build`` folder of the TYPO3 core in the file::
+
+   Build/bamboo/src/main/java/core/AbstractCoreSpec.java
+
+When you search for ``typo3DatabaseUsername`` in this Java file you will find the definitions
+of the different database credentials.
+
+.. tip::
+
+   Be shure to have a cup of coffee, a good book or other things to be done to span the waiting time,
```
Comment (liayn) [inline Documentation/BugfixingAZ/Index.rst]: shure -> sure

File: Documentation/BugfixingAZ/Index.rst [hunk 4]
```diff
@@ -79,6 +79,56 @@ If you should encounter any problems or have questions, talk to us on Slack_ in
...
+   because the functional tests will be running a lot of time longer than the unit tests. You can expect
+   a runtime of around 45 Minutes, depending on how powerful your local machine is.
```
Comment (liayn) [inline Documentation/BugfixingAZ/Index.rst]: Minutes -> minutes

**Comment by marble** [comment]
Hi Michael, cool, I love to see that information. I could push changes back to your source. So I just checked it in and did a followup update. Please see if you like everything. It's great to see documentation is more and more useful!

**Pull request merged**